### DISCUSSION TITLE
How to redeploy CoreDNS if deleted

### DISCUSSION THREAD

**Discussion by abhishek-ch** [vitobotta/hetzner-k3s#444] [General]
How to redeploy coredns if the deployment is deleted.
I tried rerun the create cluster script but that didnt relaunch coredns

**Comment by vitobotta** [comment]
You can find the manifest used by k3s to install CoreDNS at /var/lib/rancher/k3s/server/manifests/coredns.yaml on any master.

So just SSH into a master and run `k3s kubectl apply -f /var/lib/rancher/k3s/server/manifests/coredns.yaml`.

**Reply by abhishek-ch** [reply]
Worked

**Reply by vitobotta** [reply]
Nice!
